packages/components/src/border-box-control/border-box-control-split-controls/hook.ts

```diff
@@ -30,5 +30,14 @@ export function useBorderBoxControlSplitControls(
 		return cx( styles.CenteredBorderControl, className );
 	}, [] );
 
-	return { ...otherProps, centeredClassName, className: classes };
+	const rightAlignedClassName = useMemo( () => {
+		return cx( styles.RightBorderControl, className );
+	}, [] );
```

Comment:
Found while testing this the alignment of the right border control was out of whack when on tablet or mobile viewports.